Feature/storage/dm parallel transfer

amnguye opened the pull request:
Creating a PR to look over the API of DMLib

This excludes a lot of tests for now.

amnguye commented:
Give me a second I have a lot of conflicts to resolve..

jsquire commented on sdk/eventhub/Azure.Messaging.EventHubs.Experimental/api/Azure.Messaging.EventHubs.Experimental.netstandard2.0.cs:
```diff
@@ -0,0 +1,58 @@
+namespace Azure.Messaging.EventHubs.Producer
```
Please revert the Event Hubs content.

amnguye commented on sdk/storage/.editorconfig:
```diff
@@ -0,0 +1,136 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the C:\azure-sdk-for-net\sdk\storage codebase based on best match to current usage at 12/10/2021
```
I dont even know how I added this. sigh let me fix this.

jsquire replied:
much appreciated.

amnguye commented on sdk/storage/Azure.Storage.Blobs/tests/ServiceClientTests.cs:
```diff
@@ -60,7 +60,7 @@ public void Ctor_Uri()
             var accountName = "accountName";
             var accountKey = Convert.ToBase64String(new byte[] { 0, 1, 2, 3, 4, 5 });
-            var blobEndpoint = new Uri("http://127.0.0.1/" + accountName);
+            var blobEndpoint = new Uri("http://account.dfs.core.gov.net" + accountName);//new Uri("http://127.0.0.1/" + accountName);
```
TODO: undo change

Yao725 committed:
[ServiceBus]Refresh mgmt sdk (#26284)

* refresh mgmt servicebus

* upgrade servicebus to 2021-11

* Revert "upgrade servicebus to 2021-11"

This reverts commit 61b8d9f66dac4f87ce9540ffa55d11d0f75fd255.

* show request path

* first round change

* second round change

* fix tests

* update api

Yao725 committed:
revert version (#26611)

AlexGhiondea committed:
Update MigrationGuide.md (#26540)

Fix typo

Y-Sindo committed:
Update API of `ServerlessHub<T>` according to the review meeting (#25834)

* Update API of ServerlessHub<T> according to the review meeting

* Rename `Event` -> `SignalRTriggerEvents`, `Category` -> `SignalRTriggerCategories`

* Update Azure SignalR SDK to 1.15.0

Why TFM .netcoreapp3.1 deleted: The `JsonHubProtocol` of ASP.NET SignalR uses `Newtonsoft.Json` for netcoreapp 2.1 and uses `System.Text.Json` for netcore3.1 and above. However, function host always uses `Newtonsoft.Json` to parse JSON data sent from isolated worker, and SignalR fails to serialize `Newtonsoft.Json.JObject` on .NET Core 3.1 correctly. In the past, SignalR SDK and extensions enabled `NewtonsoftJsonHubProtocol` (not available on .NET Core 2.1) on .NET Core 3.1 with isolated function workers only, so they had to use framework-sensitive conditional compilation. Since SignalR SDK 1.15.0, we have a new hub protocol accepting a `ObjectSerializer`, which is available on .NET Standard 2.0, so framework-sensitive conditional is unecessary.

* Update

* Add `SignalRConnectionAttribute` for `ServerlessHub<T>`

* Fix for SignalRConnectionAttribute

* Fix

* Remove unused using.

archerzz committed:
fix(resourcemanager): append custom `user-agent` headers (#26556)

Right now, the mgmt telemetry policy will override `user-agent` header to either default header value, or `UserAgentOverride` which is set in the generated codes of each SDK. Then the custom `user-agent` headers are all lost. That will prevent 3rd parties (like Powershell) to do their analytics.

This commit will:
1. change the telemetry policy behavior to append custom headers after the official value (either default one or set by `SDKUserAgent`), which is also a best practice in the industry
2. rename `UserAgentOverride` property to `SDKUserAgent` since we now allow append `User-Agent` header
2. add unit test and benchmark test

fix #26406

arpja committed:
Adding 2021-12-01 api-version (#26561)

* - adding 2021-12-01 api-version
- updating passive stamp version to use 2021-11-15

* - Fixing sdkinfo compile error
- Updating test case with new api-versions

* - correcting paassive package name from recoveryservicebckupCrossregionRestoe to recoveryservicebckupCrossregionRestore

* - removing unnecessary files

Yao725 committed:
Revert "revert version (#26611)" (#26613)

This reverts commit ff928a7912b0de20032372d1b719c0535a966f39.

azure-sdk committed:
Increment package version after release of Azure.IoT.DeviceUpdate (#26609)

azure-sdk committed:
Increment package version after release of Azure.Data.SchemaRegistry (#26608)

azure-sdk committed:
Increment package version after release of Azure.Monitor.Query (#26603)

Co-authored-by: nisha-bhatia <67986960+nisha-bhatia@users.noreply.github.com>

moreOver0 committed:
[AnomalyDetector] Add Code Owners (#26614)

* Increase SDK version to resolve the release issues about last version

* add code owners for AnomalyDetector

* Adding service attention rule

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

navba-MSFT committed:
(AzureCXP) fix the permissions description (#26568)

navba-MSFT committed:
(AzureCXP) fix the permissions description (#26570)

navba-MSFT committed:
(AzureCXP) fix for the permissions description (#26571)

azure-sdk committed:
remove greedy suffix on link check (#26623)

Co-authored-by: Dane Walton <dawalton@microsoft.com>

m-nash committed:
Use arm client reference instead of client context (#26452)

* changes to remove armclientoptions cloning

* changes to generated files

* updates to use provider namespace string and not pass assembly

* remove usage of mgmt pipeline builder

* update api

* temp comment out compute and network for tests

* temp update to migration guide md

* temp changes to support back compat

* revert manual changes to generated folder

* make properties protected

* add client context overload

* update api

* add overload useClientContext to tenant clean up more code

* add cached client api

* update api

* update public api to be virtual

* update to use getor add

* updates to custom code

* temp changes to support autorest changes

* more customization

* remove file

* suppress armclient

* updates to custom code to adapt to new generation

* Update AutoRest C# version to 3.0.0-beta.20220126.1

* regen with new custom code

* make ctor param ordering consistent

* more adjustments on custom code and test cases

* removing some code that is no longer needed

* remove unused method

* uncomment references to compute and network
update api and snippets

* removing more api that isn't needed anymore

* add back azure.core project to sln

* regen return type fix

* Add test cases around cached factory

* Update AutoRest C# version to 3.0.0-beta.20220126.3

* update tests

* update customization

* cache sub client diag

* update apis

* update webpubsub

* update storage

* update appconfiguration

* update cdn

* update compute

* update keyvault

* update storage

* update eventhub

* update communication

* update deviceupdate

* update servicebus

* update sqlmanagement

* update webpubsub

* update appservice

* update insights

Co-authored-by: azure-sdk <azuresdk@microsoft.com>

rohkuma-microsoft committed:
Adding code for GA version 2021-12-01 (#26539)

* Adding code for GA version 2021-12-01

vicancy committed:
[Web PubSub] Adding live test for web pubsub (#25106)

* Adding live test for web pubsub

* Add live test

* Adding arm template

* Ignore live test until it is supported by CI

* Update code snnipet

ArcturusZhang committed:
Fix duplicate files in Azure.ResourceManager.Network (#26637)

* delete everything

* regenerate everything

navba-MSFT committed:
(AzureCXP) fix the permissions description (#26567)

Commit:
Update for Compute 2021-11-01 version (#26370)

* update with api version 2021-11-01

* update with api version 2021-11-01

* update virtualmachine to virtualMachine in swagger

* update test to preserve resource group until finally block

* test updates and new recordings

* update from swagger sync

* Added SDK changes for scaling VMSS with PublicIpPrefix (patch updates) (#26031)

* Add RepairAction property to VMSS AutoRepairs feature (#25095)

* Add RepairAction property to VMSS AutoRepairs feature

* Include metadata file

* Add session record and updated test

* Update TestVMScaleSetScenarioOperations_OrchestrationService.json

* Restore files that shouldn't have been changed

* Add orchestration service record

* Delete compute_resource-manager.txt

Co-authored-by: Theodore Chang <thchan@microsoft.com>

* sdk gen (#25670)

Co-authored-by: Chase Van Buskirk <cvanbuskirk@microsoft.com>

* Added sdk changes for vmss vm customization (#25785)

* Added sdk changes for vmss vm customization

* Added sdk changes for vmss vm customization

* init (#25620)

Co-authored-by: Theodore Chang <thchan@microsoft.com>

* Initial commit (#25383)

* SDK changes for CVM for api-version 2021-11-01 (#25645)

* sdk changes for CVM

* save changes for securityType and diskSecurityprofile

Add PMK scenario

* Added recorded file

* DedicatedHost Reboot Feature SDK (#25258)

* ran autorest on azure-rest-api-specs after adding dhreboot to compute.json

* ran autorest again, since i made some swagger changes

* created test

* renamed reboot to restart

* updated swagger, and ran the generate command again to update the sdk

* restart test passing. Added recorded test as well

* rebuilt sdk with async=true, removed unnecessary line in test, and reran test after change

* Adding the new paramaters zone/placementGroupId to ForceRecoveryServiceFabricPlatformUpdateDomainWalk (#25897)

* add filter for List VMSS operations  (#25802)

* fdsaf

* generated change

* Adding VMScaleSet Flex Filter Test

* Update generate.ps1

* add recording

Co-authored-by: LexieXie <lexiexie@microsoft.com>
Co-authored-by: xielexie <94083876+xielexie@users.noreply.github.com>

* add customizations

* fix errors

* regenerated from master

* fix customization

* fix customization

* fix a test

* new test recordings

* change version name

* remove unnecessary variable

* update recording test

* fix changes

Co-authored-by: karthikka4820 <88366202+karthikka4820@users.noreply.github.com>
Co-authored-by: frank-pang-msft <92764154+frank-pang-msft@users.noreply.github.com>
Co-authored-by: Chase VanBuskirk <chasevanb@gmail.com>
Co-authored-by: Chase Van Buskirk <cvanbuskirk@microsoft.com>
Co-authored-by: Raktima Das <raktdas@microsoft.com>
Co-authored-by: kamusta-msft <56413142+kamusta-msft@users.noreply.github.com>
Co-authored-by: sukodava <78733210+sukodava@users.noreply.github.com>
Co-authored-by: ms-saypaul <67093296+ms-saypaul@users.noreply.github.com>
Co-authored-by: vbhasker-msft <93355049+vbhasker-msft@users.noreply.github.com>
Co-authored-by: avjai <34567499+avjai@users.noreply.github.com>
Co-authored-by: LexieXie <lexiexie@microsoft.com>
Co-authored-by: xielexie <94083876+xielexie@users.noreply.github.com>

JoshLove-msft committed:
Update to latest TP version (#26653)

* Update to latest TP version

* fix

JoshLove-msft committed:
Switch to test proxy (#26587)

* Switch to test proxy

* Fix session recordes

* fix

* Fix tests

* Remove excess polling requests

* Delete more requests

* delete more

* re-record ARM test

* match existing order

* Switch to test proxy

* Fix session recordes

* fix

* Fix tests

* Remove excess polling requests

* Delete more requests

* delete more

* re-record ARM test

* match existing order

* update package version

* update diskaccessoperation

* update deviceupdate

* update keyvault

* update appgateway

* missing one content length

* missing one more

Co-authored-by: m-nash <prognash@gmail.com>

fengzhou-msft committed:
Bump autorest csharp to 3.0.0-beta.20220127.5 (#26656)

* remove GenericResourceFilter ResourceListOperations

* use ProviderInfo for tenant provider

* rename management lock

* regenerate

* use common type v3

* update api

* regenerate compute, network, resources

* use common type managed service identities

* manual deduplicate

* regenerate SDKs

* update api

* update ClientDiagnostics field in manual code

* fix storage resources resourcemanager cosmosdb

* fix RbacManagementClient

* fix ManagementPipelineBuilder

* fix endpoint

* resolve build failure

* deduplicate resources

* update api for deviceUpdate

* fix cdn tests

* fix deviceupdate tests

* fix legacy track2 packages with custom code

* fix testcommon

Co-authored-by: chunyu3 <chunyu@microsoft.com>
Co-authored-by: Yao Kou <yaokou@microsoft.com>

amnguye committed:
Update logger information

heaths committed:
Fix Microsoft.Extensions.Azure reference in README (#26664)

The README mistakenly referred to the package as Microsoft.Extensions.Azure.Core (no ".Core").

azure-sdk committed:
Update matrix doc header titles for replace/modify/append (#26668)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

heaths committed:
Add troubleshooting guide for Key Vault (#26636)

* Add troubleshooting guide for Key Vault

* Add certificate- and key-specific guides

* Resolve PR feedback

* Update wording about auto-discovery

rajkumar-rangaraj committed:
Azure Monitor Exporter - Separate Trace and Logs into different classes (#26548)

* Seperate trace and logs into different classes.

* remove comment as this is a version of telemetry and not actual API.

* Remove test centralized changes.

yukun-dong committed:
add storage tests (#25532)

archerzz committed:
fix(storage): broken test case due to interface change (#26673)

1. fix broken test cases which is due to interface change
2. update test recordings

fengzhou-msft committed:
[ResourceManager] Some clean up and add changelogs to prepare for release (#26672)

Yao725 committed:
Update AutoRest C# version to 3.0.0-beta.20220128.3 (#26674)

Co-authored-by: azure-sdk <azuresdk@microsoft.com>

fengzhou-msft committed:
Release Resources 1.0.0-beta.6 (#26679)

ArthurMa1978 committed:
Refresh KeyVault, ConnectedVMwarevShpere, StoragePool mgmt. SDK (#26680)

* Rename Resource -> KeyVaultResource, GetMhsmPrivateLinkResource -> GetMhsmPrivateLinkResources

* Refresh KeyVault, ConnectedVMwarevShpere, StoragePool

azure-sdk committed:
Increment version for resources releases (#26681)

Increment package version after release of Azure.ResourceManager.Resources

ArthurMa1978 committed:
Release Azure.ResourceManager.Compute-beta.6 (#26683)

* Rename Resource -> KeyVaultResource, GetMhsmPrivateLinkResource -> GetMhsmPrivateLinkResources

* Release Azure.ResourceManager.Compute-beta.6

ArthurMa1978 committed:
Release Azure.ResrouceManager.EdgeOrder 1.0.0-beta.1 (#26682)

* Rename Resource -> KeyVaultResource, GetMhsmPrivateLinkResource -> GetMhsmPrivateLinkResources

* Release Azure.ResourceManager.EdgeOrder 1.0.0-beta.1

ArthurMa1978 committed:
fix release date (#26686)

azure-sdk committed:
Increment version for connectedvmwarevsphere releases (#26684)

Increment package version after release of Azure.ResourceManager.ConnectedVMwarevSphere

azure-sdk committed:
Increment version for storagepool releases (#26685)

Increment package version after release of Azure.ResourceManager.StoragePool

ArthurMa1978 committed:
remove connectedvmware files to solve case-sensitive file name issue (#26687)

azure-sdk committed:
Increment version for compute releases (#26690)

Increment package version after release of Azure.ResourceManager.Compute

ArthurMa1978 committed:
Add back ConnectedVMwarevShpere code (#26691)

* remove connectedvmware files to solve case-sensitive file name issue

* Add back ConnectedVMwarevShpere code

azure-sdk committed:
Increment version for keyvault releases (#26689)

Increment package version after release of Azure.ResourceManager.KeyVault

archerzz committed:
feat(cosmosdb): bump track2 mgmt sdk to 1.0.0-beta.3 (#26694)

1. bump api version to `2021-10-15`
2. fix test case and update test recordings

archerzz committed:
feat(storage): bump track2 mgmt SDK 1.0.0-beta.6 (#26692)

1. bump api version to `2021-08-01`
2. update test recordings

ArthurMa1978 committed:
Fix changelog to release Azure.ResourceManager.Storage and CosmosDB (#26698)

* remove connectedvmware files to solve case-sensitive file name issue

* Fix changelog for release Storage and CosmosDB

PramodValavala-MSFT committed:
Update comments for Storage Delete methods to clarify behavior (#26403)

chidozieononiwu committed:
Fix broken links in repo (#26665)

* Fix broken links in repo

* Update sdk/identity/Azure.Identity/CHANGELOG.md

Co-authored-by: Christopher Scott <chriscott@hotmail.com>

Co-authored-by: Christopher Scott <chriscott@hotmail.com>

vishweshbankwar committed:
Add metrics exporter (#26651)

* draft

* updates

* minor changes

* refactor

* remove test file

* add sdk version

* refactor

* rmv using

* refactor

* rmv using

nickdepinet committed:
Azure Lab Services 2021-11-15-preview mangement api version bump (#26628)

* update csharp sdk to 2021-11-15-preview

* update tests

* change back to 4.0.0 version for release on nuget

Co-authored-by: Nick Depinet <ndepinet@microsoft.com>

JoshLove-msft committed:
Fully migrate more libraries to test proxy (#26703)

* Fully migrate more libraries to test proxy and re-record tables tests

* Add helper methods

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.CosmosDB (#26699)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Storage (#26701)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.EdgeOrder (#26706)

Aashish93-stack-zz committed:
[DigitalTwins] Add Code Owners (#26718)

AhmedLeithy committed:
[Question Answering Authoring] Activated deployment tests and regenerated client (#26697)

* Regenerated client

* Activated deploy tests and recorded sessions

* Applied RestClient rename

* Added TODOs for reverting slowdown bug workaround

* Added transform to edit docstring "Update -> Add"

jaschrep-msft committed:
OpenRead Avoid Unnecessary Rebuffer (#26667)

* stream changes

* tests

* compile-include extensions from auto-rest

* mock behavior

Co-authored-by: jschrepp-MSFT <41338290+jschrepp-MSFT@users.noreply.github.com>

nisha-bhatia committed:
Add TextAnalyticsAudience to Azure.AI.TextAnalytics (#26583)

kinelski committed:
[FormRecognizer] Generating from 2022-01-30 swagger, implementing currency field, adding tags dict (#1562) (#1563) (#1567) (#26731)

JoshLove-msft committed:
API updates for Avro encoder (#26734)

* API updates for Avro encoder

* Remove setter

amnguye committed:
Changed threadpool method to use inherit TransferScheduler

seanmcc-msft committed:
Enabled STG 81 live tests (#26443)

Mohit-Chakraborty committed:
[Search] Add Azure Machine Learning Skill (#26597)

* Update generated code to add `AmlSkill`

* Update public API

Add AmlSkill to RoundtripAllSkills test

* Update session record for RoundtripAllSkills test

* Add transforms to rename properties
Expose appropriate constructors

* Take model rename changes from REST spec
Make required properties as get-only
Change type of Location -> Azure.Core.AzureLocation
Change type of ResourceId -> Azure.Core.ResourceIdentifier

* Mark `AzureMachineLearningSkill.Location` as nullable

* Update changelog

* Refactor ctors on AmlSkill
Add hyperlinks on comments

* Redefine properties to support (de)serialization
Define internal-only variations of 2 properties so the generated (de)serialization code works without being manually overridden.

amnguye committed:
Remove previous implementation of getting job details, and combine into one

azure-sdk commented:
API changes have been detected in `Microsoft.Azure.WebJobs.Extensions.SignalRService`. You can review API changes [here](https://apiview.dev/Assemblies/Review/13383038cab94f1fb0c3bb4a9c2e65e1?diffOnly=True&diffRevisionId=e3e7bb50cc6b44fb990f6d87133605d2)

amnguye committed:
Removing mistake of committed files

chunyu3 committed:
10-min shippable package automation (#26098)

* Dotnet SDK automation

* dotnet SDK automation

* update swagger regex

* dataplane template

* dataplan template

* dataplane template

* update solution and project file template

* 10min package

* add alpine env setup

* use namespace as parameter

* rename resourceProvider to service

* update quickstart doc

* remove management automation script

* resolve comments

* add experiement project into solution

* support multiple input files

* resolve comments

* update script location

* update script

* move dataplane template under sdk/template-dpg folder

* update quickstart doc

* support multiple authentication method, AADToken and AzureKey

* remove unused scripts

* add guide to set security

* update links in the quickstart doc

* refine quickstart doc

* refine quickstart doc

* use list for steps of package generation script

* refine quickstart doc

* refine quickstart doc

* update quickstart doc

* format quickstart markdown doc

* move template to eng folder

* move template to eng folder

* refine dotnet dataplane template

* update README of Samples

* use Azure.Core Azure.Core.Experimental package

* update link in QuickStart

* resolve comments of Quickstart doc

* sample api

* resolve comments

* add code fence

* add customization best prastice

* remove test-resources from template

* update input-file in autorest.md

* remove scope of suppression

* update quickstart doc

* updates to quickstart doc

* fix break

* fix

Co-authored-by: Anne Thompson <annelo@microsoft.com>

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2608 (#26747)

* Get ms alias from github identity.

* Update Update-DocsMsMetadata.ps1

Co-authored-by: sima-zhu <sizhu@microsoft.com>
Co-authored-by: Sima Zhu <48036328+sima-zhu@users.noreply.github.com>

vishweshbankwar committed:
Offline Storage [Part 1] (#26626)

* draft

* rename

* revert nuget config

* temp

* package update

* revert nuget change

* try catch for storage init

* fix bug

* refactor test

* refact

* fix build error

* add check for storage

jsquire committed:
[Fabric Bot] Reflect CODEOWNERS for Digital Twins (#26749)

The focus of these changes is to update the "Service Attention" contacts
for Digital Twins to reflect the most recent updates to CODEOWNERS.

m-nash committed:
Add parameter to forwards client calls to support APIs that cache results (#26750)

* support testing APIs that cache results

* add test case to validate true and false

* add comment on new property
fix back compat issue if you are using an older client where the property didn't exist

* update to bool since this is an OR against the class value not an override to it.

jsquire committed:
[Messaging Extensions] Update README add package (#26757)

The focus of these changes is to update the README and remove the pre-release
version referenced by the instructions for adding the package to a project.

chidozieononiwu committed:
Exclude generated files from policheck analysis (#26720)

m-nash committed:
Fix Azure.Core project references (#26756)

* update core project references

* update tests in dns until we finish this project

* missed one reference

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2605 (#26746)

m-nash committed:
move test cases to azure.core.tests (#26768)

xin9le committed:
Fixes argument name for ArugmentNullException (#26770)

kinelski committed:
[FormRecognizer] Added Language detection feature (#26767)

christothes committed:
Fix contributor profile link (#26773)

annelo-msft committed:
remove missed file (#26784)

christothes committed:
Docs for RetryOptions.MaxDelay (#26778)

* Docs for RetryOptions.MaxDelay

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2673 (#26776)

azure-sdk committed:
Send authenticated request to query release tags (#26774)

Co-authored-by: Praveen Kuttappan <praveen.kuttappan@gmail.com>

JoshLove-msft committed:
Support IgnoredQueryParameters with Test Proxy (#26795)

* Support IgnoredQueryParameters with Test Proxy

* empty list check

nisha-bhatia committed:
wip (#26786)

azure-sdk committed:
Support github user namespaces for codespaces environments (#26801)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

m-nash committed:
Regenerate management libraries to support full instrumentation during tests (#26645)

* sample update to have full instrumentation

* remove unused recordings

* support testing APIs that cache results

* update to arm core to adopt full instrumentation

* update codegen

* few more updates

* fix generation issue

* update core tests

* update to async usecase

* add instance wrapper for static getters on armclient

* update custom code to adapt to new generation
regen sdk
update tests and api

* add backcompat name change to update autorest and update custom code to use new property

* revert change

* fix custom file ctor

* update api

* resourcemanager changes

* update appconfiguration

* update cdn

* update compute

* update connectedvmsphere

* update cosmos

* update deviceupdate

* update communication

* update edge order

* update eventhub

* update extendedlocation

* update keyvault

* update network

* update resources

* update servicebus

* update sql

* update storage

* update storagepool

* update webpubsub

* update appservices

* update autorest version

* update resourcemanager api and enum name change

kinelski committed:
[FormRecognizer] Added BuildMode feature (#26788)

jsquire committed:
[Event Hubs] Add missing metadata to PartitionContext (#26737)

The focus of these changes is to add metadata about the associated Event Hub
instance to the `PartitionContext` used by the consumer and processor clients.
These fields are present in Java, Python, and JavaScript but are missing in
.NET for some reason.

JoshLove-msft committed:
Fix deserialization issues in system events (#26805)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2685 (#26806)

christothes committed:
Make canonicalizedResource lower-case (#26802)

* Make canonicalizedResource lower-case

amnguye committed:
Update to Job Plan header

danielortega-msft committed:
Update codeowners for phone numbers sdk (#26813)

azure-sdk committed:
Update AutoRest C# version (#26812)

* Update AutoRest C# version to 3.0.0-beta.20220204.1

* update vault tests

* update apis

* update keyvault snippet

Co-authored-by: m-nash <prognash@gmail.com>

JoshLove-msft committed:
Trim recording size of KV ARM tests (#26817)

AntonioVT committed:
[ContainerRegistry] SDK Update for container registry, consume 2021-09-01 stable API (#26546)

m-nash committed:
fix uri property that isn't a uri (#26820)

Jingshu923 committed:
[DataFactory]Added new features into 5.2.0 (#26819)

* [DataFactory]Added new features into 5.1.0

* update

* add one change

* [DataFactory]Added new features into 5.2.0

amnguye committed:
Updating manager detail of API

amnguye committed:
Add abstract resumeJob to base storagetransfermanager

jsquire committed:
[CODEOWNERS] Git Hub and Bot Rules (#26816)

The focus of these changes is to add explicit owners for the `.github` folder
in the repository, containing items related to Git Hub integrations and the
bot rules applied to the repo.

seanmcc-msft committed:
updated storage changelogs (#26821)

jsquire committed:
[Event Hubs] Function Bindings Tests (#26829)

* [Event Hubs] Function Bindings Tests

The focus of these changes is to stabilize tests `EventHub_InitialOffsetFromEnqueuedTime`
and `EventHub_PartitionKey` which have been intermittently unstable during
nightly runs.

* Lowering timeout, as test execution is non-parallel

Commit:
[Identity] Make ClientAssertionCredential public (#26620)

* [Identity] Make ClientAssertionCredential public

* updating api spec

* adding tests

* adding test recordings

* updating api spec

* addressing feedback

* rerecording all tests for new sanitization updates

* removing superfluous project file entry

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2691 (#26830)

* Update prepare-release parameter documentation

* Apply suggestions from code review

Co-authored-by: Cala Zubair <60439897+czubair@users.noreply.github.com>

Co-authored-by: Wes Haggard <Wes.Haggard@microsoft.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>
Co-authored-by: Cala Zubair <60439897+czubair@users.noreply.github.com>

JoshLove-msft committed:
SR API updates (#26781)

* SR API updates

* Add back virtual

* rename options

* Update sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

* Update ref docs

* PR fb

* Change log

* Fix reference

* snippets

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

azure-sdk committed:
Increment version for storage releases (#26835)

dpokluda committed:
SDK sample for .NET client library. (#26725)

* SDK sample for .NET client library.

* Remove unnecessary command line examples

* Adding Directory.Build.props.

* Update sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeployUpdate/DeployUpdate.csproj

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

* Update sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeployUpdate/DeployUpdate.csproj

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

* Update sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeployUpdate/DeployUpdate.csproj

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

* Updating package references

* Update target framework

* Apply suggestions from code review

* Update sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/GetUpdate/GetUpdate.csproj

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

* Update sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeployUpdate/DeployUpdate.csproj

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

* Apply suggestions from code review

* Apply suggestions from code review

* Update sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/Directory.Build.props

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

christothes committed:
Support ResourceId for user defined MSI (#26595)

* support ResourceId for MSI

azure-sdk committed:
Update AutoRest C# version (#26831)

* Update AutoRest C# version to 3.0.0-beta.20220207.1

* update api

* temp changes leaving back compat list method in until autorest changes can go in

* update armresource to remove back commpat method

* update compute test

Co-authored-by: m-nash <prognash@gmail.com>

seanmcc-msft committed:
Modified StorageBearerTokenChallengeAuthorizationPolicy to use scope r… (#26717)

heaths committed:
Prepare Question Answering release (#26748)

rajkumar-rangaraj committed:
Azure Monitor Exporter - Add Exception Telemetry (#26670)

* Add Exception Telemetry

* Formatted message.

* Add TODO.

vighatke committed:
Features/iothub/2021 07 02 controlplane (#26771)

* Regenerate from latest swagger API version 2021-07-02
* Adding tests
* updating version

Co-authored-by: David R. Williamson <drwill@microsoft.com>
Co-authored-by: VIBHA GHATKE <vighatke@microsoft.com>

heaths committed:
Prepare for Key Vault beta release (#26793)

* Regenerate Administration on latest 7.3-preview

* Re-enable backup/restore tests

Resolves #26122

* Add KeyReleasePolicy.Immutable property

Resolves #26364

* Enable SKR for both KV and MHSM

Resolves #22750

* Resolve build issues

* Change ISO8601 durations to strings

* Resolve PR feedback

tjegbejimba committed:
[Resources] add generated files from Subscriptions 2021-01-01 (#26615)

* initial subscription autogeneration

* fix nextpagelink

* update resource-manager.txt

* add updated test records

* update assemblyfile version

priyjain358 committed:
Track2 Management SDK for StackHCI (#26408)

nisha-bhatia committed:
Add FormRecognizerAudience and DocumentAnalysisAudience to Azure.AI.FormRecognizer (#26741)

JoshLove-msft committed:
Prepare for release (#26844)

AhmedLeithy committed:
[CLU] Prepare package for release (#26853)

* Updated changelog

* Ran prepare-release

* Resolve PR feedback

Co-authored-by: Heath Stewart <heaths@outlook.com>

danielortega-msft committed:
Skip update capabilities live test (#26759)

This prevents the update capabilities test from running in the live test pipeline, which caused an issue with parallel test execution updating the same phone number.

annelo-msft committed:
Fixes to CHANGELOG to release Azure.Core.Experimental (#26859)

* update changelog

* disable flaky tests

JoshLove-msft committed:
Support skipping recording (#26847)

* Support skipping recording

* Re-migrate form recognizer

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2686 (#26862)

azure-sdk committed:
Increment version for search releases (#26865)

Increment package version after release of Azure.Search.Documents

AriZavala2 committed:
GA release for NetworkTraversal (#26758)

* GA new version of swagger autorest code and fixing serialization/deserialization issue

* Allow expires on to be nullable

* Fix code checks

* Fix version string in csproj file

* Add testing for Serialization/deserialization

* Add new recordings

* Remove nullable of ExpiresOn

* Update PR

* Update date

danielortega-msft committed:
Add missing codeowners (#26860)

nisha-bhatia committed:
Create CredscanProcess.md (#26825)

jaschrep-msft committed:
Progress Reporting Blob Downloads (#26838)

* added progress to base download

* tests

* downloadTo and tests

* exportapi

* changelog

* null safety

Co-authored-by: jschrepp-MSFT <41338290+jschrepp-MSFT@users.noreply.github.com>

azure-sdk committed:
Increment version for core releases (#26867)

Increment package version after release of Azure.Core.Experimental

christothes committed:
Handle AzurePowerShellCredential not logged in scenarios on non-windows (#26868)

* handle login issues on non-windows

annelo-msft committed:
Update packages.data.props and references to Azure.Core.Experimental for Feb release (#26866)

* update references to Azure.Core.Experimental for Feb release

* update packages.data.props

benbp committed:
Install git into identity keyring test container image (#26722)

* Install git into identity keyring test container image

* Reference globals in identity dockerfile to skip security analysis

* Bump dotnet patch versions and checksums

* Install latest git into ubuntu keyring container

* Bump identity ubuntu keyring testing image

* Skip dotnet logger install in container jobs (don't require msbuild)

azure-sdk committed:
Increment version for cognitivelanguage releases (#26870)

* Increment package version after release of Azure.AI.Language.QuestionAnswering

* Increment package version after release of Azure.AI.Language.Conversations

heaths committed:
Update Key Vault CHANGELOGs (#26874)

azure-sdk committed:
Print example docker commands after building stress image (#26873)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

fengzhou-msft committed:
Release Azure.ResourceManager.StackHCI (#26850)

azure-sdk committed:
Increment version for keyvault releases (#26877)

* Increment package version after release of Azure.Security.KeyVault.Keys

* Increment package version after release of Azure.Security.KeyVault.Administration

m-nash committed:
draft of adding armoperation types to always return (#26808)

* draft of adding armoperation types to always return

* update namespace

* regen back for ci

* revert test cases and update api

* tweak to api to hide more implementation

* update to pass in finalstate via

* update appconfiguration

* update cdn

* update cdn snippets

* update communication

* update compute

* update connectedvmwarevsphere

* remove unnecessary configs

* update cosmos

* update resourcemanager

* update deviceupdate

* update edgeorder

* update eventhub

* update extendedlocation

* update keyvault

* update network

* regen appconfiguration

* regen cdn

* regen communication

* regen compute

* regen connectedvmware

* regen cosmosdb

* regen deviceupdate

* regen resourcemanager

* regen edgeorder

* regen eventuhub

* regen extendedlocation

* regen keyvault

* update resources

* update servicebus

* update sql

* update storage

* update storagepool

* update webpubsub

* update appservice

* update apis

* update azurestackhci

* remove save inputs for cosmosdb

* remove save inputs from appservice

* regen removing unused operation source classes

* Update AutoRest C# version to 3.0.0-beta.20220208.1

* regen azurestackhci

Co-authored-by: azure-sdk <azuresdk@microsoft.com>

Yao725 committed:
bump cdn version (#26849)

amnguye committed:
Update event arguments to have individual ones for each case, update progress handler to basic handler,  update copy method

JoshLove-msft committed:
Bump test proxy version (#26882)

AriZavala2 committed:
Update Changelog (#26876)

jsquire committed:
[Event Hubs Function Extension] Disable Tests (#26890)

The focus of these changes is to disable tests failing due to the limitations
of data binding in the WebJobs SDK until a fix is released.

This is tracked by:
https://github.com/Azure/azure-webjobs-sdk/issues/2830

jsquire committed:
[Event Hubs] February Release Prep (#26891)

The focus of these changes is to prepare the `Azure.Messaging.EventHubs`
package for its February milestone release.

kinelski committed:
[FormRecognizer] Implemented DocumentAnalysisModelFactory (#26893)

m-nash committed:
temp changes to add in new ResourceName (#26861)

* temp changes to add in new ResourceName

* udpate api and instrumentation

* Update AutoRest C# version to 3.0.0-beta.20220208.2

* update api

* update test case reference

* update api

* update appservice test reference

Co-authored-by: azure-sdk <azuresdk@microsoft.com>

danielmarbach committed:
ServiceBus Changelog changes (#26897)

JoshLove-msft committed:
Don't propagate ObjectDisposedException when cancellation was requested (#26822)

azure-sdk committed:
Increment version for eventhub releases (#26900)

Increment package version after release of Azure.Messaging.EventHubs

jsquire committed:
[Event Hubs Processor] February Release Prep (#26899)

The focus of these changes is to prepare the `Azure.Messaging.EventHubs.Processor`
package for its February milestone release.

JoshLove-msft committed:
Fix shared folder (#26902)

seanmcc-msft committed:
Fixed Recording mode in progress handler tests (#26903)

sima-zhu committed:
Generate new baseline (#26892)

sima-zhu committed:
Adding cred scan step in PR validation and Ci build (#26527)

JoshLove-msft committed:
Remove Other Changes section (#26906)

azure-sdk committed:
Add max value to DeleteAfter of 168 hours (#26840)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

danielmarbach committed:
Processor improvements (#26904)

* Use the right overload for the ContinueWith in the receiver manager make sure the continuation is actually using ExecuteSynchronously

* Update change log

* Remove array allocation from linked token source creation

* Update changelog

christothes committed:
Log WWW-Authenticate header by default (#26887)

azure-sdk committed:
Add catch-all parameter to test resources script (#26898)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

mpodwysocki committed:
Update Azure Notification Hubs SDK to 2.3.4-preview (#26337)

* Updating for 2.3.4

* Version bump

* Updating with latest commands

* Updating AssemblyInfo

mikeharder committed:
Use azsdk-pool in aggregrate-reports.yml (#26907)

azure-sdk committed:
Preprocess the target branch + Remove deleted files (#26915)

Co-authored-by: sima-zhu <sizhu@microsoft.com>

JoshLove-msft committed:
Update Schema Registry dependency version (#26914)

archerzz committed:
feat(resourcemanager): replace `Scope` with `ArmEnvironment` (#26648)

* feat(resourcemanager): replace `Scope` with `ArmEnvironment`

1. Create an unsealed enum `ArmEnvironment` which represents various Azure cloud environments. `ArmEnvironment` contains two properties right now:
  1.1 `BaseUri` which is the base URI of the management endpoint.
  1.2 `Audience` which is the authentication audience.
  1.3 Also there is a handy `Scope` property which is deduced from `Audience`
2. Change `ArmClientOptions.Scope` to `ArmClientOptions.Environment`
3. Simplify constructors of `ArmClient` to the following 3:
  3.1 `cred`
  3.2 `cred, subscriptionId`
  3.3 `cred, subscriptionId, options`

resolve #26635

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20220209.2 (#26883)

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventHubs.Processor (#26908)

azure-sdk committed:
Increment version for communication releases (#26901)

Increment package version after release of Azure.Communication.NetworkTraversal

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid (#26912)

azure-sdk committed:
Increment package version after release of Azure.Messaging.ServiceBus (#26913)

LeszekKalibrate committed:
remove duplication (#26917)

jaschrep-msft committed:
`PartitionedDownloader` Fixes (#26787)

* copy changes

* revert to copyto

* changelog

* changelog fix

* updated with pulled changelog rev

* changed thrown exception type

Co-authored-by: jschrepp-MSFT <41338290+jschrepp-MSFT@users.noreply.github.com>

tyrielv committed:
Fix typo in MigrationGuide.md (#26927)

kinelski committed:
[Form Recognizer] Updated docs and added prebuilt-read sample (#26926)

jsquire committed:
[CODEOWNERS] Fix duplication for Schema Registry (#26936)

* [CODEOWNERS] Fix duplication for Schema Registry

Removing the duplicate entry and updating the service contact.

JoshLove-msft committed:
Prepare for release (#26935)

vishweshbankwar committed:
Refactor rest client (#26871)

* Refactor trackasync

* storage null case

* resolve few pr comments

* revert storagehelper

* add using instead of try finally

* refactor

* rmv using

* newline

* minor

* fix dispose

* resolve pr comment

Commit:
Adding Azure.Identity.BrokeredAuthentication package (internals visible) (#26772)

* Adding Azure.Identity.BrokeredAuthentication for broker authentication support

* adding changelog

* adding readme and tests

* address feedback

* add unit test

* fb

azure-sdk committed:
Fixed the prefix of the target branch (#26943)

Co-authored-by: sima-zhu <sizhu@microsoft.com>

zedy-wj committed:
Update text analytics to enable live testing in sovereign clouds for multiple services (#26823)

* Sovereign clouds tests with Text Analytics

* Update test-resources.json.disabled

remove invalid changes

* Update tests.yml

Remove Canary cloud

Co-authored-by: zedy <zedy@wicresoft.com>

kinelski committed:
[FormRecognizer] Fixed CHANGELOG release date (#26948)

Commit:
Gallery 2021-10-01 release (#26658)

* generated with new version reference for gallery

* re-geenerate with communityGallery and sharedGallery removed from new version

* re-record tests

* Gallery 10-01 sdk (#25941)

* save

* add test for community gallery publishing

* update

* save (#25940)

* updated

* add customization

* fix, record tests

* update version

* re gen

* update

* Update GalleryApplicationVersion_CRUD_Tests.json

Co-authored-by: kangsun-ctrl <69279251+kangsun-ctrl@users.noreply.github.com>

azure-sdk committed:
Increment version for formrecognizer releases (#26949)

Increment package version after release of Azure.AI.FormRecognizer

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2740 (#26952)

amnguye committed:
Removed base DM models, made base event args, made protected ctor StorageTransferManager

leigharubin committed:
Generate 2021-10-15 SDK and update tests (#26752)

JoshLove-msft committed:
Expose ReleaseSession in SB extension (#26950)

* Expose ReleaseSession in SB extension

* PR fb

JoshLove-msft committed:
Fix extension method name (#26969)

Commit:
[Identity] Updating changelog for 1.6.0-beta.1 release (#26967)

azure-sdk committed:
Update AutoRest C# version (#26944)

* Update AutoRest C# version to 3.0.0-beta.20220210.1

jaschrep-msft committed:
parent navigators added to clients (#26785)

* parent navigators added

* fix

Co-authored-by: jschrepp-MSFT <41338290+jschrepp-MSFT@users.noreply.github.com>

JoshLove-msft committed:
Use value comparison for byte arrays (#26968)

* Use value comparison for byte arrays

* Fix enumeration error

kinelski committed:
Re-enable tests (24552) (#26966)

heaths committed:
Add proxy environment variable documentation (#26929)

* Add proxy environment variable documentation

Resolves #23774

* Resolve PR feedback

JialinXin committed:
[WebPubSub][Functions] Respect user's cancellation token in output binding. (#26920)

Fix #26704.

Y-Sindo committed:
[SignalR] Update `Microsoft.Azure.SignalR.Management` version to 1.15.1 (#26963)

azure-sdk committed:
Increment package version after release of Azure.Identity (#26974)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.SignalRService (#26993)

christothes committed:
Changes for discussion to the PR Template (#26631)

* Changes to the PR Template

jsquire committed:
[Legacy Smoke Tests] Resolve Dependency Issue (#26984)

Bumping the version for `Microsoft.Azure.Amqp` to avoid triggering a
message about downgrading versions.

heaths committed:
Add CredScan exemptions for Key Vault (#26930)

danielortega-msft committed:
[Communication] - PhoneNumbers - Implement a phone number pool for live test pipelines (#26910)

* Implement phone number pool for live tests

* Skip capabilities live tests for INT pipeline jobs

Also, rename matrix configuration file

* Update matrix config file reference in test project

* Use relative path to matrix configuration file

Co-authored-by: Ben Broderick Phillips <ben@benbp.net>

* Select phone numbers based on test agent

This avoids the need to make code changes when the test matrix or the phone numbers pool is modified.

This approach requires that the test agent identifier is explicitly defined as an environment variable in the job matrix in order to have a consistent way to get that information at runtime across different framework and OS versions.

Co-authored-by: Ben Broderick Phillips <ben@benbp.net>

danielmarbach committed:
TrackPublishHandlerAsActiveAsync closure and synchronous invocation hint (#26986)

* TrackPublishHandlerAsActiveAsync closure and synchronous invocation hint

* Update sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubBufferedProducerClient.cs

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

* Update sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubBufferedProducerClient.cs

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

* Properly fix whitespace, sigh

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

jsquire committed:
[Event Hubs] Minor Performance Tweaks (#26983)

* [Event Hubs] Minor Performance Tweaks

The focus of these changes is to apply some of the performance-oriented
tweaks made in Service Bus to the Event Hubs clients.   Included are:

- Attempt to retrieve AMQP objects synchronously before calling `GetOrCreateAsync`

- Remove LINQ from the `AmqpMessageConverter`

- Change the internal batch `AsEnumerable<T>` to `AsList<T>` in order to avoid
  casting costs and have `Count` available to right-size transform collections.

- Use the two item overload when creating a linked token source to avoid
  allocating an unnecessary array.  _([ref](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Threading/CancellationTokenSource.cs#L736-L739))_

m-nash committed:
Resolve autorest conflicts for 20220214.1 (#26996)

* Update AutoRest C# version to 3.0.0-beta.20220214.1

* update api

* update cdn tests

* update deviceupdate tests

* update network tests

* add placeholder tests for subscription tag methods

Co-authored-by: azure-sdk <azuresdk@microsoft.com>

danielmarbach committed:
Service Bus trigger should ignore scheduled/deferred messages (#25127)

* Better tests for QueueTime

* Test to verify scheduled messages are ignored

* Ignore scheduled messages for scaling and queue time

* Add additional verification test for metrics

* Extract into method with better name

* Add ability to extract the state from the annotated message

* Test helper

* Modifiers

* Adjust tests

* Change implementation

* Switch tests to assume peeking

* Naive peeking

* Switch to single peek and storing sequence number

* Make sure to not run into actual admin client operations that time out and slow down the tests

* More test scenarios for new algorithm

* New algorithm as discussed

* Better variable names

* Fix styling

* Upgrade to Azure.Messaging.ServiceBus 7.6.0

* Review comments

* Update sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusScaleMonitor.cs

Co-authored-by: JoshLove-msft <54595583+JoshLove-msft@users.noreply.github.com>

danielortega-msft committed:
Fix test pipeline script format (#26997)

jsquire committed:
[Fabric Bot] Fix rule typo (#27004)

The focus of these changes is to fix a typo in a comment left by the bot when
a stale pull request is detected.

Commit:
[Identity] Update ci.yml to include BrokeredAuthentication package (#27000)

danielmarbach committed:
Remove LINQ allocations for all batched sends (#26911)

* Remove the LINQ allocations for all outgoing messages that are batched

* Array.Empty for my eyes

* Mock the transport batch properly

* Cleanup no longer needed using

* Switch case over the backing store

* Use directly the concrete collection

* Use concrete type where possible for Diagnostic

Avoid additional array allocation when receiving

* Use the concrete list type also for scheduled messages including the count

* Switch back to readonly since that seems the better strategy even though it will be slower to enumerate

* Fix the event source tests

Commit:
Update CODEOWNERS for Azure IoT SDK (#27005)

JoshLove-msft committed:
Bump NUnit adapter version (#26998)

christothes committed:
Handle 429 responses for InsertOverPartitionLimit test (#27002)

* handle 429 with Recording.Delay

vishweshbankwar committed:
Storage transmission evaluator (#26976)

* draft

* add samplesize

* update timer

* refactor names

* resolve pr comments

ArcturusZhang committed:
Update test cases in deviceupdate (#27010)

* fix the tests in deviceupdate

* re-record deviceupdate tests

Co-authored-by: Yao Kou <yaokou@microsoft.com>

vishweshbankwar committed:
remove unnecessay usings (#27012)

vishweshbankwar committed:
Fix naming violations (#27011)

* fix naming violations

* fix failing tests

jsquire committed:
[Fabric Bot] Updating IoT Owners (#27006)

The focus of these changes is to add entries for the IoT-related service
contacts, based on recent updates to CODEOWNERS.

heaths committed:
Suppress some KV issues in sovereign clouds (#27009)

christothes committed:
Ignore flaky test (#27023)

AikoBB committed:
Add a backoff mechanism for the CommunicationTokenCredential (#25031)

* Added a backoff mechanism for the CommunicationTokenCredential

rajkumar-rangaraj committed:
Azure Monitor Exporter - Update LogRecord StateValues (#26857)

* StateValues

* Fix statevalues

* Feedback

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2754 (#27029)

* Add configs

* Update eng/common/pipelines/templates/steps/credscan.yml

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

* Adding commnets

* wrong order

* Update credscan.yml

* Update credscan.yml

Co-authored-by: sima-zhu <sizhu@microsoft.com>
Co-authored-by: Sima Zhu <48036328+sima-zhu@users.noreply.github.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

m-nash committed:
Update body param for Patch methods (#26961)

* appconfiguration changes

* azurestackhci changes

* cdn changes

* Revert "cdn changes"

This reverts commit ea86313815361c089d7cf356eaa902e4d8574ab8.

* update appconfiguration

* update azurestackhci

* update azurestackhci

* update compute

* update connectedvmwarevsphere

* update cdn

* update cosmos

* update deviceupdate

* update edgeorder

* update keyvault

* update edgeorder

* update cdn

* update compute

* update eventhub

* update extendedlocation

* update keyvault

* update network

* update resourcemanager

* update resources

* update servicebus

* update sqlmanagement

* update storage

* update storagepool

* update appservices

* update tagresource

* update sqlmanagement

* update api

* update snippets

* revert local changes

* update cdn

* update compute

* update connectedvmwarevsphere

* update deviceupdate

* update resources

* update sqlmanagement

* update appconfiguration

* update azurestackhci

* update cdn

* update communication

* update compute

* update connectedvmwarevsphere

* update cosmosdb

* update deviceupdate

* update resourcemanager

* update edgeorder

* update eventhub

* update extendedlocation

* update keyvault

* update network

* update resources

* update servicebus

* update sqlmanagement

* update storage

* update storagepool

* update webpubsub

* update appservices

* update api

* Update AutoRest C# version to 3.0.0-beta.20220214.3

* TryGetApiVersion change in ArmClient

* bump autorest

* update api

* remove duplicate files

Co-authored-by: azure-sdk <azuresdk@microsoft.com>
Co-authored-by: Yao Kou <yaokou@microsoft.com>

HarveyLink committed:
Update swagger_to_sdk_config.json (#26992)

Fix to avoid x-ms-identifiers issue

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20220215.1 (#27041)

tjegbejimba committed:
privatelink client generation (#27035)

* privatelink client generation

* update assemblyversions

azure-sdk committed:
Add warning/exit for Az.Resources 5.3.0 service principal provisioning error (#26977)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

kinelski committed:
[FormRecognizer] Ignoring Neural build mode test when running live (#27043)

seanmcc-msft committed:
Fixed ShareFileClient.StartCopy() IgnoreReadonly (#27028)

vishweshbankwar committed:
Refactor transmitter (#27061)

* draft

* remove transmitfromstorage

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 2758 (#27027)

* tooling updates. allow install but no run of test proxy, prepending the path properly, etc
* update targeted versions of the test-proxy

Co-authored-by: scbedd <45376673+scbedd@users.noreply.github.com>

m-nash committed:
update to 20221602.2 (#27063)

JoshLove-msft committed:
Migrate storage tests to test proxy (#26662)

* Migrate storage tests to test proxy

* Remove unused method

* Update empty request body recordings

* Reset upstream URI on each retry

* remove pct encoding from header values

* Datalake session file fixes

* more decoding

* datalake passing .net 6

* Datalake passing on all platforms

* Files passing

* .NET framework sanitization

* Avoid nesting proxy transport

* Fix sanitizer

* Undo unnecessary re-recording

* revert more

* fix

Yao725 committed:
Remove inheritance relation of AccountUpdateOptions to InstanceUpdateOptions (#27038)

m-nash committed:
Update AutoRest C# version to 3.0.0-beta.20220216.3 (#27066)

Co-authored-by: azure-sdk <azuresdk@microsoft.com>

m-nash committed:
refresh snippets based on API changes (#27064)

* refresh snippets based on API changes

* remove one var

* update snippet for var

* update RI ctor consistency

* remove section on tryget RI methods

* remove getallasync

* update comment on getall

HarveyLink committed:
Update Directory.Build.Mgmt.props (#27071)

azure-sdk committed:
Update AutoRest C# version (#27074)

* Update AutoRest C# version to 3.0.0-beta.20220217.1

* refresh api

* fix test case compile failure in storage

Co-authored-by: ArcturusZhang <dapzhang@microsoft.com>

amisi01 committed:
Add restorable apis for table & gremlin. (#26724)

* Generated code for 2021-11-15-preview release

* Added tests and updated version

* Test Run changes

* Updating version and sanitizing test.

* Update PR with materialized views.

* Updating CEK changes.

* Fix test

* re-record the failed test

Co-authored-by: Yao Kou <yaokou@microsoft.com>

jaschrep-msft committed:
Nullable CopyStatus on BlobProperties (#26946)

* fixed

* changelog

* exportapi

Co-authored-by: jschrepp-MSFT <41338290+jschrepp-MSFT@users.noreply.github.com>

JoshLove-msft committed:
Relax entity name validation for the case of a pre-formatted subscription path (#27070)

* Relax entity name validation for the case of a pre-formatted subscription

* typo

* PR fb

automagicaly committed:
[phone-numbers sdk] Marks as 'deprecated' one of the PhoneNumbersMode… (#26730)

* [phone-numbers sdk] Marks as 'deprecated' one of the PhoneNumbersModelFactory methods

* [phone-numbers sdk] add missing dependency

* [phone-numbers sdk] Updates public API

* [phone-numbers sdk] Updates CHANGELOG

* [phone numbers sdk] Updates Changelog to be more descriptive

* [phone-numbers sdk] Revert CHANGELOG update

azure-sdk committed:
Add logic to generate docs.ms ToC (#27045)

Co-authored-by: Daniel Jurek <djurek@microsoft.com>

amnguye committed:
Changed Directory Download to DownloadTo, added overwrite options, updated internal pipeline transfer for directoryclient

amnguye committed:
Merge branch 'main' into feature/storage/dm-parallel-transfer

amnguye committed:
change string to uri for local paths, remove unncessary things from blob job properties

amnguye committed:
WIP - changing job details out, added more champ scenarios regarding progress tracking
